scss/style.scss

```diff
@@ -0,0 +1,6 @@
+$heading: red;
+
+//h1 heading color
+h1 {
+  color: $heading;
+}
```

Comment:
Insert a final newline here and in the rest of the files. You can also enable this option in VSCode by adding "files.insertFinalNewline": true to your settings.json file